docs/topics/testing/tools.txt

```diff
@@ -1133,24 +1133,59 @@ don't need to test multi-database activity.
 As an optimization, Django only flushes the ``default`` database at
 the start of each test run. If your setup contains multiple databases,
 and you have a test that requires every database to be clean, you can
-use the ``multi_db`` attribute on the test suite to request a full
-flush.
+use the ``databases`` attribute on the test suite to request extra databases
+to be flushed.
 
 For example::
 
-    class TestMyViews(TestCase):
-        multi_db = True
+    class TestMyViews(TransactionTestCase):
+        databases = {'default', 'other'}
 
         def test_index_page_view(self):
             call_some_test_code()
 
-This test case will flush *all* the test databases before running
-``test_index_page_view``.
+This test case will flush the ``default`` and ``other`` test databases before
+running ``test_index_page_view``. It's also possible to use the `'__all__'`
+constant to specify that *all* the test databases must be flushed.
 
-The ``multi_db`` flag also affects into which databases the
-:attr:`TransactionTestCase.fixtures` are loaded. By default (when
-``multi_db=False``), fixtures are only loaded into the ``default`` database.
-If ``multi_db=True``, fixtures are loaded into all databases.
+The ``databases`` flag also affects into which databases the
+:attr:`TransactionTestCase.fixtures` are loaded. By default fixtures are only
+loaded into the ``default`` database. If ``databases`` is defined, fixtures are
+loaded into all the specified databases.
+
+    .. versionchanged:: 2.2
```

Comment:
It looks unusual to mark something changed in 2.2 that was added in 2.2.

Reply:
Right. I guess this whole block is now unnecessary given the following text below?

> By default only the ``default`` database will be wrapped in a transaction
> during ``TestCase``'s execution and queries against other databases will be
> disallowed in order to prevent state leaking between tests.